src/date-picker/src/DatePicker.tsx

```diff
@@ -466,6 +466,9 @@ export default defineComponent({
       ...uniVaidation,
       ...dualValidation
     })
+    provide('datePickerSlot', {
```

Comment:
Provide it using `datePickerInjectionKey`. There's already a provide in the file. Use that is enough.